tests/test_pigeon.py

```diff
@@ -307,7 +319,8 @@ async def test_format_metadata_for_ia_item(
                 ],
                 "osf_registry": "OSF Registries",
                 "osf_registration_schema": "Open-Ended Registration",
-                "source": "http://localhost:5000/g752b",
```

Comment:
These values changed while the tests were offline, so they are adjusted now that they are back.